scdblfinder/v1.12.0/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM pgc-images.sbgenomics.com/d3b-bixu/scrna_qc:v1.0.1
+
+LABEL maintainer="Miguel Brown (brownm28@chop.edu)"
+LABEL description="scDblFinder - use qc docker as base image as usage in workflow is wrapped in an R markdown notebook. Heavy stuff"
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update && apt install -y libbz2-dev libxt-dev
```

Comment:
is update enough without upgrade?

Came across this at https://askubuntu.com/questions/94102/what-is-the-difference-between-apt-get-update-and-upgrade ?

> apt-get update updates the list of available packages and their versions, but it does not install or upgrade any packages.
apt-get upgrade actually installs newer versions of the packages you have. After updating the lists, the package manager knows about available updates for the software you have installed. This is why you first want to update.

Reply:
Um, probably can't hurt. Update for sure is needed as over time the list of available packages gets stale. The autoremove tip at the end might be useful to clean up in the end. I can add it on and see

Reply:
Ok, added. Seems fine